resource/plugins/unreal/python/loader/importers/unreal_abc_animation_loader_importer.py

```diff
@@ -0,0 +1,109 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+import os
+
+import unreal
+
+from ftrack_connect_pipeline_unreal import plugin
+from ftrack_connect_pipeline_unreal.constants.asset import modes as load_const
+from ftrack_connect_pipeline_unreal.utils import (
+    custom_commands as unreal_utils,
+)
+import ftrack_api
+
+
+class UnrealAbcAnimationLoaderImporterPlugin(
+    plugin.UnrealLoaderImporterPlugin
+):
+    load_modes = load_const.LOAD_MODES
+
+    plugin_name = 'unreal_abc_animation_loader_importer'
+
+    def run(self, context_data=None, data=None, options=None):
+        '''Load Alembic animation file pointed out by collected *data*, with *options*.'''
+
+        # Build import task
+
+        task = unreal.AssetImportTask()
+
+        paths_to_import = []
+        for collector in data:
+            paths_to_import.extend(collector['result'])
+
+        component_path = paths_to_import[0]
+
+        task.filename = component_path
+
+        asset_version_entity = self.session.query(
+            'AssetVersion where id={}'.format(context_data['version_id'])
+        ).first()
+        import_path = (
+            '/Game/'
+            + unreal_utils.get_context_relative_path(
+                self.session, asset_version_entity['task']
+            )
+            + context_data['asset_name']
+        )
+
+        task.destination_path = import_path.replace(' ', '_')
```

Comment:
IS this safe? we are replacing spaces to underscore in the path, so the path will not be valid anymore... right?

Reply:
This is the destination path, Unreal does not support space in folders/file names.